src/main/java/blackjack/domain/card/Card.java

```diff
@@ -0,0 +1,52 @@
+package blackjack.domain.card;
+
+import java.util.Objects;
+
+public class Card {
+
+    private final Type type;
+    private final Denomination denomination;
+
+    public Card(Type type, Denomination denomination) {
+        this.type = type;
+        this.denomination = denomination;
+    }
+
+    public String findTypeName() {
+        return type.getName();
+    }
+
+    public String findDenominationName() {
+        return denomination.getName();
+    }
+
+    public int findDenominationValue() {
+        return denomination.getNumber();
+    }
+
+    public boolean isAce() {
+        return denomination == Denomination.ACE;
+    }
+
+    public boolean isNotAce() {
+        return denomination != Denomination.ACE;
```

Comment:
```suggestion
        return !isAce()
```
는 어떤가요 ㅎㅎ 메모리상 아주 미세한 낭비는 있겠지만 
개인적으로 가독성은 더좋다고 생각해요.